aleph/views/collections_api.py

```diff
@@ -258,19 +265,26 @@ def bulk(collection_id):
     if entityset is not None:
         entityset = get_entityset(entityset, request.authz.WRITE)
 
-    # This will disable checksum security measures in order to allow bulk
+    # This will disable (if False) checksum security measures in order to allow bulk
     # loading of document data:
     safe = get_flag("safe", default=True)
     # Flag is only available for admins:
     if not request.authz.is_admin:
         safe = True
 
+    # This will disable (if True) values validation for all types of all entities / properties
+    # (will pass cleaned=True to the model.get_proxy() in the aleph/logic/processing.py)
+    cleaned = get_flag("cleaned", default=False)
```

Comment:
Slightly concerned about the fact that the two flags `cleaned` and `safe` do related things but behave in opposite ways when set to `true`:

* `cleaned=true`: Assume that the data **can be trusted** and contains only valid values.
* `safe=true`: Assume that the data **can NOT be trusted**, so perform additional security measures.

Not sure though if there is a better alternative and probably not a huge problem as this is mostly relevant in edge cases. A `validate` flag is used in some other endpoint. It’s has a slightly different behavior though (`validate=true` means that Aleph will return an error instead of silently discarding invalid values).

Reply:
Oh, maybe the flag could be `clean` instead of `cleaned` (and default to `true`)?

* `clean=true` means that the data cannot be trusted and that the data should be cleaned from invalid values.
* `safe=true` means that the data cannot be trusted and that file checksums should be removed.

Reply:
done!